amqp/src/main/scala/akka/stream/alpakka/amqp/scaladsl/AmqpFlow.scala

```diff
@@ -61,14 +43,14 @@ object AmqpFlow {
    *
    * This stage materializes to a `Future` of `Done`, which can be used to know when the Flow completes,
    * either normally or because of an amqp failure.
+   *
+   * @param settings default value is 20 for `bufferSize` and 100ms for `confirmationTimeout`
```

Comment:
Risky to have the values in the docs, AFAICS you changed the buffer size to 10 already.

Reply:
Right, good catch 😉